src/services/item/plugins/thumbnail/service.ts

```diff
@@ -19,7 +20,7 @@ export class ItemThumbnailService {
     this.itemService = itemService;
   }
 
-  async upload(actor: Member, repositories: Repositories, itemId: string, file: UploadedFile) {
+  async upload(actor: Member, repositories: Repositories, itemId: string, file: WriteStream) {
```

Comment:
Not sure if the type makes sense here, how can an uploadable file come from a writable stream? Where does the constraint come from? @morganridel